nemo_curator/modules/semantic_dedup/clusteringmodel.py

```diff
@@ -47,13 +47,15 @@ def add_dist_to_cents(
     return df
 
 
+# Clustering module
 class ClusteringModel:
     def __init__(
         self,
         id_column: str,
         max_iter: int,
         n_clusters: int,
-        clustering_output_dir: str,
+        cache_dir: Optional[str] = None,
+        clustering_save_loc: str = "clustering_results",
```

Comment:
We deprecate `clustering_output_dir` in favor of `cache_dir` and `clustering_save_loc` for `ClusteringModel`, which matches the logic in the `SemDedup` class.